File: SY_20190727.java
/* 
*  DB Script Tool
*  Java - 2019-07-27 10:59:57
*  
*  MODEL CLASSES FOR SY DATABASE
*/ 



/* SY.java -------------------------------------*/
package com.package.SY.model; 

import java.util.*;
import java.sql.*;
import java.text.*;

/* 
*  Java - Model Class - SY.SY
*  2019-07-27 10:57:35
*/ 
public class SY {

    // private members 
    private String m_SY;


    /**
    * Constructor
    * 
    * Example: 
    * SY mySY = new SY();
    */
    public SY() {
        //--
    }

    /**
    * Constructor
    * 
    * Example: 
    * SY mySY = new SY( val1, val2,.. );
    */
    public SY(String SY) {
        this.setSY(SY);
    }


    /**
    * Getters and Setters
    */

    public String getSY() {
        return this.m_SY;
    }

    public void setSY(String SY) {
        this.m_SY = SY;
    }



    /**
    * Methods
    */

    @Override
    public String toString() {
        return "";
    }

}

File: SY_20190727.php
<?php
/* 
*  DB Script Tool
*  PHP - 2019-07-27 10:59:41
*  
*  MODEL CLASSES FOR SY DATABASE
*/ 
?>



<?php 
/* SY.php -------------------------------------*/
//namespace SY/model; 

/* 
*  PHP - Model Class - SY.SY
*  2019-07-27 10:57:35
*/ 
class SY {

    // private members 
    private $m_SY;


    /**
    * Constructor
    * 
    * Example: 
    * $mySY = new SY();
    */
    public function __construct() {
        //--
    }

    /**
    * Constructor
    * 
    * Example: 
    * $mySY = SY::WithParams( val1, val2,.. );
    */
    public static function WithParams($SY) {
        $instance = new self();
        $instance->setSY($SY);
        return $instance;
    }


    /**
    * Getters and Setters
    */

    public function getSY() {
        return $this->m_SY;
    }

    public function setSY($SY) {
        $this->m_SY = $SY;
    }



    /**
    * Methods
    */

    public function __toString() {
        return "";
    }

}
?>